cmd/k8s-netperf/k8s-netperf.go

```diff
@@ -128,21 +129,33 @@ func main() {
 		if strings.Contains(nc.Profile, "STREAM") {
 			metric = "Mb/s"
 		}
-		var serverIP string
+		var netperfServerIP string
+		var iperfServerIP string
 		var service bool
 		service = false
 		if nc.Service {
 			service = true
-			serverIP = s.Service.Spec.ClusterIP
+			netperfServerIP = s.NetperfService.Spec.ClusterIP
+			iperfServerIP = s.IperfService.Spec.ClusterIP
 		} else {
-			serverIP = s.Server.Items[0].Status.PodIP
+			netperfServerIP = s.Server.Items[0].Status.PodIP
+			iperfServerIP = s.Server.Items[0].Status.PodIP
 		}
 		if !s.NodeLocal {
+
+			// netperf Result
 			npr := result.Data{}
 			npr.Config = nc
 			npr.Metric = metric
 			npr.Service = service
 			npr.HostNetwork = true
+
+			// iperf3 Result
+			ipr := result.Data{}
+			ipr.Config = nc
+			ipr.Metric = metric
+			ipr.Service = service
+			ipr.HostNetwork = true
```

Comment:
Just a minor nit. These expressions can be optimized/reduced to:
```suggestion
			// iperf3 Result
			ipr := result.Data{
			  Config: nc,
			  Metric: metric,
			  Service: service,
			  HostNetwork: true
			}
```
So you don't have to assign each field individually